src/View/Helper/FormHelper.php

```diff
@@ -1024,6 +1024,10 @@ public function input($fieldName, array $options = [])
         $options = $this->_parseOptions($fieldName, $options);
         $options += ['id' => $this->_domId($fieldName)];
 
+        if (strtolower($options['type']) === 'input') {
+                throw new Exception(sprintf('Input is an invalid type option for input field %s', $fieldName));
```

Comment:
You haven't added `use` statement for `Exception`. You can use `RuntimeException` instead.

`throw new RuntimeException(sprintf('Invalid type "input" used for field "%s"', $fieldName));`